### DISCUSSION TITLE
Videos don't play on Windows Media Player

### DISCUSSION THREAD

**Discussion by Atledbet11** [akiver/cs-demo-manager#875] [Q&A] [answered]
Hi, I have a question regarding the output generated from the video section.

It appears that the video will not play on windows media player, so it is rather difficult to rewatch the clips.
I can watch them if I open my video editing software, but having to do this every time I make a clip is obnoxious when I have many to make.

Is this a configuration thing I can solve with settings? Or is this just a bug?

Here is a picture of my settings:
![image](https://github.com/akiver/cs-demo-manager/assets/18058118/1b36196a-e295-4f15-96bc-ec950350f4da)

**Comment by akiver** [comment, marked as answer]
Same as https://github.com/akiver/cs-demo-manager/issues/519.
You must use a video player supporting more video/audio codecs such as [VLC](https://www.videolan.org/).
I added a doc about it [here](https://cs-demo-manager.com/docs/guides/video#i-cant-play-the-video).
